### PR TITLE
chore(project): clean up error handling (DSP-1619)

### PR DESCRIPTION
**Pull request by mdelez**
resolves DSP-1619

### CODE REVIEW

File: services/admin/backend/api/handler/project.go [hunk 1]
```diff
@@ -60,43 +59,42 @@ func createProject(service project.UseCase) http.Handler {
 		// convert input strings to value objects
 		sc, err := valueobject.NewShortCode(input.ShortCode)
 		if err != nil {
-			w.WriteHeader(http.StatusInternalServerError)
+			w.WriteHeader(http.StatusBadRequest)
```
Comment (mpro7) [inline services/admin/backend/api/handler/project.go:62]: What's the benefit of change `StatusInternalServerError` to `StatusBadRequest`?

File: services/admin/backend/entity/project/error.go [hunk 2]
```diff
@@ -19,17 +19,29 @@ package project
...
 var ErrNoPropertiesChanged = errors.New("no new value for any property provided")
 
 //ErrCannotBeDeleted cannot be deleted
 var ErrCannotBeDeleted = errors.New("cannot be deleted")
 
 //ErrProjectHasBeenDeleted project has been marked as deleted
 var ErrProjectHasBeenDeleted = errors.New("project has been marked as deleted")
+
+//ErrShortCodeAlreadyExists provided short code already exists
+var ErrShortCodeAlreadyExists = errors.New("provided short code already exists")
```
Comment (mpro7) [inline services/admin/backend/entity/project/error.go:47]: Shouldn't all the errors start with capital letter?

**Review by mpro7 — APPROVED** [review]
Looks ok. Please answer my comments.

File: services/admin/backend/api/handler/project.go (hunk 1 above)
Comment (mpro7) [reply services/admin/backend/api/handler/project.go:62]: That's logic. I've already approved this PR, just wanted to know the background of the change, which seems to be right.

Comment (subotic) [reply services/admin/backend/api/handler/project.go:62]: @mdelez I agree.

**Pull request merged**